.github/workflows/ci.yml

```diff
@@ -1,32 +1,53 @@
 name: CI
 
-on: [push, pull_request]
+on: 
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
 
 jobs:
   build:
     runs-on: ubuntu-latest
-    services:
-      postgres:
-        image: postgres:11
-        env:
-          POSTGRES_USER: postgres
-          POSTGRES_PASSWORD: postgres
-          POSTGRES_DB: ci_db_test
-        ports:
-          - 5432:5432
-        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
     strategy:
       matrix:
-        node: ['8', '10', '12', '14', '16', '17']
-        os: [ubuntu-latest, windows-latest, macos-latest]
+        #node: ['8', '10', '12', '14', '16', '17']
+        #os: [ubuntu-latest, windows-latest, macos-latest]
+        # let's make it a little bit simple for now
+        node: ['14', '17']
+        os: [ubuntu-latest]
     name: Node.js ${{ matrix.node }} (${{ matrix.os }})
     steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - uses: actions/checkout@v2
+
+      - name: Setup Vertica
+        env:
+          VERTICA_CE_URL: "https://vertica-community-edition-for-testing.s3.amazonaws.com/XCz9cp7m/vertica-11.1.1-0.x86_64.RHEL6.rpm"
+        run: |
+          git clone https://github.com/jbfavre/docker-vertica.git
+          curl $VERTICA_CE_URL --create-dirs -o docker-vertica/packages/vertica-ce.latest.rpm
+          docker build -f docker-vertica/Dockerfile.centos.7_9.x --build-arg VERTICA_PACKAGE=vertica-ce.latest.rpm -t jbfavre/vertica docker-vertica
+          docker images
+          docker run -d -p 5433:5433 jbfavre/vertica
+          sleep 60
```

Comment:
Why not just use the official [vertica-ce Docker image](https://hub.docker.com/r/vertica/vertica-ce)?

```
docker run -d -p 5433:5433 vertica/vertica-ce:11.1.1-0
```

Reply:
I think either the [Vertica CE](https://hub.docker.com/r/vertica/vertica-ce) (Ubuntu 18) or [Vertica Kubernetes image](https://hub.docker.com/r/vertica/vertica-k8s) (CentOS 7.9) image would be the way to go since they are both still actively managed and updated.  We use the Vertica Kubernetes image for testing the Spark connector, although either one is fine since we are simply accessing the Vertica server and not the OS.

That docker-vertica repo is quite old and hasn't been updated in years, and it requires additional dependencies and steps when a simple Docker run would suffice.

Reply:
@blackeyepanda You can use the [Vertica CE](https://hub.docker.com/r/vertica/vertica-ce) (Ubuntu 18) or [Vertica Kubernetes](https://hub.docker.com/r/vertica/vertica-k8s) (CentOS 7.9) image. vertica-python and vertica-sql-go just haven't done the switch to these latest images.

Reply:
Looks like ci.yml has been updated to use Vertica CE.  Has this conversation been resolved?

Reply:
yep, I will use Vertica CE for this pr, then we can change the vertica docker image python and go client use in the future, I will create PR for them lately.